### ISSUE TITLE
Update typescript-coding-guidelines page (include section to use tslint-to-eslint-config)

### ISSUE THREAD

**Issue by atrianac** [adidas/adidas.github.io#21]
TSLint was deprecated as is described in Palantir blog post [TSLint in 2019](https://medium.com/palantir/tslint-in-2019-1a144c2317a9). On [typescript-coding-guidelines ](https://adidas.github.io/contributing/typescript-coding-guidelines) page should include a section to promote the use of ESLint over TSLint.

Note: There is a specific [package](https://github.com/adidas/js-linter-configs/tree/master/packages/eslint-config-adidas-typescript) which use typescript-eslint build by adidas.

**Comment by RecuencoJones** [comment]
You are right @atrianac, we will update when possible, thanks for the heads up!
